exercises/run-length-encoding/example.rs

```diff
@@ -0,0 +1,45 @@
+use std::cmp;
+
+pub fn encode(input: &str) -> String {
+    input
+        .chars()
+        .fold((String::new(),' ',0,1), |(mut acc, last, last_n, pos), c| {
```

Comment:
spaces after the commas in the tuple? To be consistent with the fact that there are spaces after commas in other places